app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,63 @@
+class AlbumsController < ApplicationController
+
+  def get_album
```

Comment:
I love your get_album method! You could also set this as a private helper method that you call in a before acton at the top of your controller. Example: ```before_action :get_album, only: [:edit, :show, :update]```